grails-app/controllers/org/broadinstitute/orsp/AuthenticatedController.groovy

```diff
@@ -328,7 +328,7 @@ class AuthenticatedController implements Interceptor, UserInfo, ExceptionHandler
     }
 
     def issueIsForbidden(issue) {
-        return permissionService.issueIsForbidden(issue, getUser().userName, isAdmin(), isViewer())
+        getUser() ? permissionService.issueIsForbidden(issue, getUser().userName, isAdmin(), isViewer()) : null
```

Comment:
looks like issueIsForbidden  is a boolean function, it is OK to return a null ?

Reply:
You're right, I was trying to maintain the value it was returning before, but after a deeper look it is as you say, a boolean function. I've changed the return value to `false`.